tic-tac-toe.js

```diff
@@ -1,7 +1,71 @@
-function TicTacToe() {
+var turn = 0;
+var icon = 'X';
+var player = 'Player 1';
 
+function TicTacToe() {
+  this.board = ["cell 0", "cell 1","cell 2","cell 3","cell 4","cell 5","cell 6","cell 7","cell 8"];
 }
 
-TicTacToe.prototype = {
-  
-}
+
+
+TicTacToe.prototype.getBoard = function() {
+  return this.board;
+};
+
+TicTacToe.prototype.markBoard = function() {
+  var self = this;
+  $("td").bind("click", function() {
+    var index = 0;
```

Comment:
If you prefer writing out `index` and/or if it makes more sense to you, feel free to stick with it. But if you want to, feel free to use `i` instead -- it's pretty common in the industry, and most (all?) people will recognize `i` as being the same thing as `index`